workspaces/sonarqube/.changeset/proud-jobs-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-sonarqube-backend': minor
```

Comment:
This is a breaking change for people using Basic Auth and would need to be marked as one.

Reply:
Also is there any reason why both basic auth and bearer auth?

Reply:
I tested it and the same basic auth tokens work using bearer auth instead on both Server and Cloud, so this shouldn't be a breaking change unless you have some very strange proxies enforcing the type of Authorization header.

Reply:
@niallmccullagh, can you follow up? Can you test this? Would be good to get you feedback, I'm assuming you use this plugin.